int32 a
int32 b
int32 client_id
---
int32 prime_count
int32 client_id